infra/charts/feast/values-dataflow-runner.yaml

```diff
@@ -0,0 +1,100 @@
+feast-core:
+  gcpServiceAccount:
+    enabled: true
+  application-override.yaml:
+    feast:
+      stream:
+        options:
+          bootstrapServers: <kafka_sevice_load_balancer_ip_address_1:31090>
+      jobs:
+        runner: DataflowRunner
+        options:
+          project: <google_project_id>
+          region: <dataflow_regional_endpoint e.g. asia-east1>
+          zone: <google_zone e.g. asia-east1-a>
+          tempLocation: <gcs_path_for_temp_files e.g. gs://bucket/tempLocation>
+          network: <google_cloud_network_name>
+          subnetwork: <google_cloud_subnetwork_path e.g. regions/asia-east1/subnetworks/mysubnetwork>
+          maxNumWorkers: 1
+          autoscalingAlgorithm: THROUGHPUT_BASED
+          usePublicIps: false
+          workerMachineType: n1-standard-1
+          deadLetterTableSpec: <bigquery_table_spec_for_deadletter e.g. project_id:dataset_id.table_id>
+        metrics:
+          host: <prometheus_statsd_exporter_load_balancer_ip_address>
+
+feast-online-serving:
+  store.yaml:
```

Comment:
Some stragglers here. I think this configuration is out of date, along with the batch serving example. `store.yaml` doesn't exist any more afaik.